spec/abi.dd

```diff
@@ -581,6 +585,7 @@ $(GNAME Parameter2):
     $(B J) $(GLINK Type)     $(GREEN // out)
     $(B K) $(GLINK Type)     $(GREEN // ref)
     $(B L) $(GLINK Type)     $(GREEN // lazy)
+    $(B Nk) $(GLINK Type)     $(GREEN // return)
```

Comment:
I think that if we keep the current split between `Parameter` and `Parameter2`, `Nk` should go next to `scope` in `Parameter`.

Reply:
(Because both `scope` and `return` can be combined with `out` and `ref`.)

Reply:
Yes, I missed. Thanks for pointing out!

Reply:
No worries, and excuse me for not being specific the first time I suggested adding `Nk` to the docs!